File: src/main/java/no/nav/domain/pensjon/regler/beregning/Sertillegg.java
package no.nav.domain.pensjon.regler.beregning;

import no.nav.domain.pensjon.regler.kode.YtelsekomponentTypeCti;

/*
 * Særtillegget
 */

public class Sertillegg extends Ytelseskomponent {

    

    /**
     * Prosentsatsen
     */
    private double pSats_st;

    private int PREG_orginalBrutto;


    public Sertillegg() {
        super();
        ytelsekomponentType = new YtelsekomponentTypeCti("ST");
    }


}

File: src/main/java/no/nav/pensjon/regler/domain/beregning/Sertillegg.kt
package no.nav.pensjon.regler.domain.beregning

import no.nav.pensjon.regler.domain.kode.YtelsekomponentTypeCti

class Sertillegg : Ytelseskomponent() {
    /**
     * Prosentsatsen
     */
    var pSats_st = 0.0
    var pREG_orginalBrutto = 0
    var pREG_orginalBruttoPerAr = 0.0


    init {
        ytelsekomponentType = YtelsekomponentTypeCti("ST")
    }

}